modules/cli/src/main/scala/scala/cli/commands/export0/Export.scala

```diff
@@ -211,14 +232,27 @@ object Export extends ScalaCommand[ExportOptions] {
       project.print(System.out)
     }
     else {
-      val sbtVersion = options.sbtVersion.getOrElse("1.10.1")
+      val sbtVersion                  = options.sbtVersion.getOrElse("1.10.1")
+      val defaultMavenCompilerVersion = options.mvnVersion.getOrElse(Constants.mavenVersion)
+      val defaultScalaMavenCompilerVersion =
+        options.mvnScalaVersion.getOrElse(Constants.mavenScalaCompilerPluginVersion)
+      val defaultMavenExecPluginVersion =
+        options.mvnExecPluginVersion.getOrElse(Constants.mavenExecPluginVersion)
 
       def sbtProjectDescriptor0 =
         sbtProjectDescriptor(options.sbtSetting.map(_.trim).filter(_.nonEmpty), sbtVersion, logger)
 
       val projectDescriptor =
         if (shouldExportToMill)
           millProjectDescriptor(options.shared.coursierCache, options.project, logger)
+        else if (shouldExportToMaven)
+          mavenProjectDescriptor(
+            defaultMavenCompilerVersion,
+            defaultScalaMavenCompilerVersion,
+            defaultMavenExecPluginVersion,
+            Nil,
+            logger
+          ) // todo: do we need extraSettings?
```

Comment:
```suggestion
          ) 
```
NIT: dev leftover? what do you mean by `extraSettings` here?

Reply:
I kept it to support for future extension to add more attributes/settings to the pom. I removed the comment, but I think, it is better to leave the argument there as it is for adding more options in future. wdyt?